packages/@lwc/engine-server/src/renderer.ts

```diff
@@ -38,26 +45,14 @@ function createElement(name: string, namespace?: string): HostElement {
     };
 }
 
-const registry: Record<string, CustomElementConstructor> = create(null);
```

Comment:
another big simplification, we only need to return the constructor that when new'd creates the fake element obj in server, no more constructor, or mimic of the DOM.